src/DotNetWorker.Grpc/GrpcWorker.cs

```diff
@@ -212,8 +214,7 @@ internal static WorkerInitResponse WorkerInitRequestHandler(WorkerInitRequest re
             response.Capabilities.Add("TypedDataCollection", bool.TrueString);
             response.Capabilities.Add("WorkerStatus", bool.TrueString);
             response.Capabilities.Add("HandlesWorkerTerminateMessage", bool.TrueString);
-            response.Capabilities.Add("HandlesInvocationCancelMessage", bool.TrueString);
```

Comment:
This looks like an accidental removal from merge resolution.

Reply:
Yes good catch!